src/chai/ArrayManager.cpp

```diff
@@ -81,6 +81,20 @@ ArrayManager::ArrayManager()
 #endif
 }
 
+void ArrayManager::finalize()
```

Comment:
@davidbeckingsale whatever we decide to do in Umpire I'll do here as well.
https://github.com/LLNL/Umpire/pull/84#discussion_r267993850